src/pages/resources/fundamentals/measurements-and-positioning/index.md

```diff
@@ -0,0 +1,37 @@
+# Measurements and positioning
+
+All items and objects in InDesign are positioned on the page according to the measurements you choose. It's helpful to know how the InDesign coordinate system works and what measurement units it uses.
+
+### Coordinates
+InDesign, like every other page layout and drawing program, uses simple, two-dimensional geometry to set the position of objects on a page or spread. The horizontal component of a coordinate pair is referred to as x; the vertical component as y. You can see these coordinates in the Transform panel or Control when you select an object using the Selection tool.
+
+<!-- TODO Add screenshots of transform panel -->
```

Comment:
@anoopvalomkot  Please add screenshot of transform panel.
(Or else create a Jira ticket)

Reply:
![image](https://github.com/AdobeDocs/uxp-indesign/assets/113042047/0161e3b4-6ab2-4d5a-8f05-0acf5d8ef1eb)

Reply:
Can you please share the screenshot via email so that the resolution doesnt get distorted?